schedule/virt_autotest/install_guest_on_slem_kvm_host.yaml

```diff
@@ -25,26 +29,21 @@ conditional_schedule:
     DO_NOT_INSTALL_HOST:
       0:
         - "{{bootup}}"
-        - installation/welcome
-        - installation/scc_registration
-        - installation/ntp_config_settings
-        - installation/user_settings_root
-        - installation/resolve_dependency_issues
-        - installation/select_patterns
-        - installation/installation_overview
-        - installation/disable_grub_graphics
-        - installation/start_install
-        - installation/await_install
-        - installation/logs_from_installation_system
-        - installation/reboot_after_installation
-  install_guest:
-    SKIP_GUEST_INSTALL:
-      0:
-        - virt_autotest/unified_guest_installation
+        - "{{install}}"
   bootup:
     IPXE:
       1:
         - installation/ipxe_install
       0:
         - boot/boot_from_pxe
+  install:
+    USB_BOOT:
+      1:
```

Comment:
Will be good to keep original DVD installation for sle micro 5.5- via `USB_BOOT=0` in this yaml?

Reply:
Yes. They can add their test modules with another entry ```USB_BOOT=0```.